health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/ViewHierarchy.js

```
@@ -522,6 +536,47 @@ const ViewHierarchy = () => {
                         />
                     </div>
                 }
+                {showPopUp && (
+                  <PopUp
+                    className={"popUpClass"}
+                    footerclassName={"popUpFooter"}
+                    type={"default"}
+                    // style={{width:"70%"}}
+                    heading={t("DOWNLOAD_EXCEL_TEMPLATE_FOR_BOUNDARY")}
+                    children={[
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)